packages/components/src/components/wizard/wizard-step.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+const Step = ({ children, className }) => {
```

Comment:
Can we rename this to `WizardStep` to have a consistent filename and default export?